app/controllers/pictures_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PicturesController < ApplicationController
+
+  before_action :authenticate_user!
+
+  def index
+    @pictures = Picture.all
+  end
+
+  def new
+    @picture = Picture.new
+  end
+
+  def create
+    Picture.create_with_user(picture_params, current_user)
+    redirect_to pictures_path
+  end
+
+  def destroy
+    @picture = Picture.find(params[:id])
+    if @picture.user == current_user
+      @picture.destroy
+      flash[:notice] = 'Picture deleted successfully'
+      redirect_to pictures_path
+    else
+      flash[:notice] = 'Sorry, you may only delete your own picture'
+      redirect_to pictures_path
```

Comment:
Style/IdenticalConditionalBranches: Move `redirect_to pictures_path` out of the conditional.